shared/templates/template_ANSIBLE_sysctl

```diff
@@ -5,8 +5,8 @@
 # disruption = medium
 - name: Ensure sysctl {{{ SYSCTLVAR }}} is set to {{{ SYSCTLVAL }}}
   sysctl:
-    name: {{{ SYSCTLVAR }}}
-    value: {{{ SYSCTLVAL }}}
+    name: '{{{ SYSCTLVAR }}}'
+    value: '{{{ SYSCTLVAL }}}'
```

Comment:
Is there any reason, why you used single quotation marks here?

Reply:
I'm crazy.

Reply:
Should be fixed.